uswds/uswds-site
# USWDS-Site: Update HTMLProofer

## Summary
Updated the HTMLProofer script configuration to match the gem's updated settings, and remediated the errors found after updating the configuration. 

## Related issue
Closes #1734

## Preview link
[Federalist ](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/al-htmlproofer/)

## ~Expected errors~
~:warning: Note: In this PR, errors are still expected on the following pages:~
- ~./_site/about/security/index.html~
- ~./_site/documentation/developers/index.html~
- ~./_site/getting-started/showcase/all/index.html~

~This content is pulled in from files in the USWDS repo and are resolved in PR https://github.com/uswds/uswds/pull/4902.~ 
This PR was merged and these errors are no longer expected.

## Testing and review
To test:
1. Open `al-htmlproofer` branch
1. Remove `_site` directory if it exists
1. Run `npm start`
1. Run `npm run proof` and confirm that the only errors presented are from the pages listed above in the "Expected errors" section of this PR

To review: 
- Confirm that updated urls work and point to the correct location.
- Confirm that the deleted links need to be removed (there is no alternate location the link can point to).

## Reference
- [htmlproofer upgrade guidance](https://github.com/gjtorikian/html-proofer/blob/main/UPGRADING.md)
- [htmlproofer command line options](https://github.com/gjtorikian/html-proofer/blob/main/bin/htmlproofer)
- [htmlproofer command line instructions](https://github.com/gjtorikian/html-proofer#using-on-the-command-line)
---

Before opening this PR, make sure you’ve done whichever of these applies to you:
